src/build/settings/base.json

```diff
@@ -2,5 +2,5 @@
     "app_name": "NTU Learn Downloader",
     "author": "Kee Wan Ting",
     "main_module": "src/main/python/main.py",
-    "version": "0.0.2"
```

Comment:
i will document later what the release process should be like. Basically, there will be a `release` branch that will track `master` where public releases will be based on. Bumps to the version should be done with PRs that will immediately be merged into the `release` branch